src/app/shared/code-editor/code-editor.component.ts

```diff
@@ -47,6 +47,7 @@ export class CodeEditorComponent implements AfterViewInit {
 
   ngAfterViewInit() {
     const aceMode = 'ace/mode/' + this.mode;
+    ace.config.set('workerPath', 'https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.4/');
```

Comment:
As an alternative, you could make this a relative path so it works on dev builds, then change .circleci/config.yml to update this for gui.dockstore.org. Or was this already the approach you tried when you said it was too much of a pain?

Advantage: Less risk of an update to Ace breaking if also we forget to update this line.
Disadvantage: Makes code between prod/local a little more different than it is already and I'm wild about changing the source code like are doing (and we may move away from that if we go full bore CloudFront).

Reply:
I find the sed to be a lot more fragile than this because we'd change file names, change how this command looks, etc. Different versions of the worker-json.js actually work, so I'm not too worried about it even if we do update ace at some point.